ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -1016,7 +1304,14 @@ const NetworksForm = ({
             'data-testid': 'network-form-ticker-input',
           }}
           value={ticker}
-          disabled={viewOnly}
+          disabled={
+            (viewOnly ||
+              isDefaultMainnet ||
+              isDefaultLineaMainnet ||
+              isDefaultLineaSepoliaTestnet ||
+              isDefaultSepoliaTestnet) &&
+            !addNewNetwork
```

Comment:
Is this the same set of conditions for the previous `disabled` usage? If so, supports use of `const`.